plugin-maven/CHANGES.md

```diff
@@ -3,6 +3,8 @@
 We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (starting after version `1.27.0`).
 
 ## [Unreleased]
+### Fixed
+* Maven plugin with `ktfmt` default style uses correct continuation indent ([#1562](https://github.com/diffplug/spotless/pull/1562))
```

Comment:
This entry should also be in `plugin-gradle/CHANGES.md` and `/CHANGES.md`

Reply:
Yes, I added them now.